processing/src/main/java/org/apache/druid/segment/QueryableIndexCursorSequenceBuilder.java

```diff
@@ -110,8 +110,10 @@ public Sequence<Cursor> build(final Granularity gran)
     );
 
     final int numRows = index.getNumRows();
-    final FilterBundle filterBundle = makeFilterBundle(bitmapIndexSelector, numRows, true);
+    final FilterBundle filterBundle = makeFilterBundle(bitmapIndexSelector, numRows);
 
+    // filterBundle will only be null if the filter itself is null, otherwise check to see if the filter
+    // can use an index
```

Comment:
If the point is to know if the filter was set or not, can we not change it to explicitly check if the filter was set instead of asking for a bundle to be made and believing that `null` from that means that the filter didn't exist?

Reply:
yea, i guess i did it this way the nullability check doesn't associate filter bundle not being null with filter not being null which makes it kind of ugly <img width="556" alt="Screenshot 2024-02-28 at 12 52 56 AM" src="https://github.com/apache/druid/assets/1577461/ce92f664-ee74-4b9f-a8d6-065167484194">